src/app/components/SearchBox/SearchBox.module.scss

```diff
@@ -21,18 +21,9 @@
     width: 100%;
   }
 
-  button {
-    padding: $spacing-xs;
-    display: flex;
-    align-items: center;
-    cursor: pointer;
-
-    &:disabled {
-      cursor: not-allowed;
-
-      img {
-        filter: invert(92%) sepia(7%) saturate(39%) hue-rotate(227deg) brightness(87%) contrast(87%);
-      }
+  &__disabled {
```

Comment:
If you want to use `disabled` as a class, then it should be written as a BEM modifier `—disabled` and not as an BEM element. But in current case it’s better to use disable attribute for these styles